### ISSUE TITLE
"elm-app build" writes incorrect location in link tags 

### ISSUE THREAD

**Issue by jtdaad** [halfzebra/create-elm-app#104]
Might be a bug, might a problem with my configuration, in which case the documentation could be updated for the correct config.

My problem: 
When I run elm-app build, the build process does not error, the problem is that in the dist/index.html file I get a "/" in front of all the links for <link> tags.  In the console I see this is causing the page to think the resources are at the root level instead of at the same folder level.  

Current built code:
```
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <title>Elm-App</title>
    <link rel="shortcut icon" href="/favicon.ico">
    <link href="/css/main.b8f4f52c.css" rel="stylesheet">
</head>

<body>
    <div id="root"></div>
    <script type="text/javascript" src="/js/main.2f80aea6.js"></script>
</body>

</html>
```

Corrected built code:
```
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <title>Elm-App</title>
    <link rel="shortcut icon" href="favicon.ico">
    <link href="css/main.b8f4f52c.css" rel="stylesheet">
</head>

<body>
    <div id="root"></div>
    <script type="text/javascript" src="js/main.2f80aea6.js"></script>
</body>

</html>
```

**Comment by halfzebra** [comment]
@jtdaad thanks for the contribution!

You can surely have relative paths, please check out the [Deploying to GitHub Pages](https://github.com/halfzebra/create-elm-app/blob/master/template/README.md#deploying-to-github-pages) section.

**Comment by halfzebra** [comment]
The issue with incorrect paths is fixed in [b33b44478e99d295b908f5e01a30ea218f04427c](https://github.com/halfzebra/create-elm-app/commit/b33b44478e99d295b908f5e01a30ea218f04427c)

It's included in the latest release.

I'm closing this issue, please feel free to reopen it if you are still experiencing it with the latest version.
